vignettes/cards.Rmd

```diff
@@ -113,98 +106,85 @@ card(
     class = "bg-dark",
     "A header"
   ),
-  markdown("Some text with a [link](https://github.com)")
+  markdown("Some text with a [link](https://github.com).")
 )
 ```
 :::
 :::
 
-## Fixed sizing
+## Scrolling contents
 
 ::: row
 ::: col-md-6
-By default, a `card()`'s size grows to accommodate the size of it's contents. Thus, if some portion of the `card_body()` contains a large amount of text, table(s), etc., consider setting a fixed `height`. And in that case, if the contents exceed the specified height, they'll be scrollable.
-:::
-
-::: col-md-6
-```{r}
-card(
-  card_header(
-    "A long, scrolling, description"
-  ),
-  card_body(
-    height = 150, 
-    lorem_ipsum_dolor_sit_amet
-  )
-)
-```
-:::
-:::
+By default, a `card()`'s size grows to accommodate the size of it's contents. Thus, if a `card_body()` contains a large amount of text, table(s), etc., you may want to specify a `height`. Although, when laying out [multiple cards](#multiple-cards), you probably shouldn't specify a `height` on the card-level (instead, the height should probably come from a containing element, like `layout_column_wrap()`).
 
-::: row
-::: col-md-6
-Alternatively, you can also set the `height` of the card to a fixed size and set `fill = TRUE` to have the `card_body()` container shrink/grow to fit the available space in a `card()`. Note that, by doing this, the _children_ of the `card_body()` aren't necessarily allowed to shrink/grow to fit the `card_body()`, which `card_body_fill()` (aka "[responsize sizing](#responsize-sizing)") is designed to do.
+Although scrolling is convenient for reducing the amount of space required to park lots of content, it can also be a nuisance to the user. To help reduce the need for scrolling, consider pairing scrolling with `full_screen = TRUE` (which adds an icon to expand the card's size to the browser window).
 :::
 
 ::: col-md-6
 ```{r}
 card(
   height = 200,
+  full_screen = TRUE,
   card_header(
     "A long, scrolling, description"
   ),
-  card_body(
-    fill = TRUE,
-    lorem_ipsum_dolor_sit_amet
+  lapply(
+    lorem::ipsum(paragraphs = 3, sentences = c(5, 5, 5)),
+    tags$p
```

Comment:
```suggestion
  lorem::ipsum(paragraphs = 3, sentences = 5)
```

`sentences` of length 1 are recycled and `lorem::ipsum()` has an `as.tags()` method now :)